python/setup.py

```diff
@@ -204,9 +204,18 @@ def get_packages(self):
 # also update the matching section of requirements/requirements.txt
 # in this directory
 if setup_spec.type == SetupType.RAY:
+    if sys.version_info >= (3, 7):
+        pandas_dep = "pandas >= 1.3"
```

Comment:
@richardliaw Note that this Pandas version constraint change is not required for the fix, but matches our version constraint in CI and is a good change; there are a lot of fixes in Pandas 1.3 that Datasets relies on.

Reply:
The Datasets fix sounds good; would prefer not to raise that everywhere across the libraries though (i.e., probably don't need it for tune)

Reply:
thank yoU!